examples/clients/elphmod/driver.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import elphmod
+
```

Comment:
Please, explain the main keywords of elphmod used here or add a doc string with the appropriate links to the elphmod documentation

Reply:
Thanks for the hint! Is the minimal docstring with the link to the documentation of the relevant module enough? (I have chosen the Markdown format because it is used in some other docstrings of the repository.)

Reply:
I would prefer to leave as Jan has it now, because describing all elphmod keywords here would be a repetition of work (that has been nicely done elsewhere).

Reply:
yes, that is enough thanks